front/src/router/routes.ts

```diff
@@ -54,6 +55,7 @@ const ctfRoute: RouteRecordRaw = {
   children: [
     {
       path: 'task/:taskId(\\d+)-:taskSlug',
+      alias: ['task/:taskId(\\d+)'],
```

Comment:
When I am visiting a URL like `http://localhost:8088/#/ctf/1/info` and then click on the tasks tab, then I get an empty screen and the following warning in the console:
![screenshot of error](https://user-images.githubusercontent.com/34778827/162575340-7abbe34b-3a36-4f21-83ca-c3dd73a10a14.png)
The URL that has the empty page is `http://localhost:8088/#/ctf/1-Midnight-Sun-CTF-2022-Quals/tasks` in my case. So this is probably due to the slug being added again to the URL.

Reply:
Ok so I can't find an easy fix for this right now. Unfortunately I will be pretty busy again this week and might only get to this next week, sorry.